### ISSUE TITLE
Warning: Invalid version react-native@0.68.1 for expo sdkVersion 45.0.0. Use react-native@0.68.2

### ISSUE THREAD

**Issue by tayfunyasar** [expo-community/expo-firebase-starter#71]
i see so many warnings.

```

Warning: Invalid version react-native@0.68.1 for expo sdkVersion 45.0.0. Use react-native@0.68.2
iOS Bundling complete 53599ms
iOS Running app on Tayfun12PM

AsyncStorage has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-async-storage/async-storage' instead of 'react-native'. See https://github.com/react-native-async-storage/async-storage
at node_modules/react-native/Libraries/Utilities/warnOnce.js:27:2 in warnOnce
at node_modules/react-native/index.js:273:12 in module.exports.get__AsyncStorage
at node_modules/@firebase/auth/dist/rn/index.js:127:71 in <global>
at node_modules/metro-runtime/src/polyfills/require.js:339:11 in loadModuleImplementation
at node_modules/firebase/auth/dist/index.esm.js:1:0 in <global>
at node_modules/metro-runtime/src/polyfills/require.js:339:11 in loadModuleImplementation
at navigation/RootNavigator.js:3:0 in <global>
at node_modules/metro-runtime/src/polyfills/require.js:339:11 in loadModuleImplementation
at App.js:4:0 in <global>
at node_modules/metro-runtime/src/polyfills/require.js:339:11 in loadModuleImplementation
at node_modules/expo/AppEntry.js:3:0 in <global>
at node_modules/metro-runtime/src/polyfills/require.js:339:11 in loadModuleImplementation
at node_modules/metro-runtime/src/polyfills/require.js:200:44 in guardedLoadModule
at http://192.168.1.125:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false&strict=false&minify=false:203844:3 in global code

[react-native-gesture-handler] Seems like you're using an old API with gesture components, check out new Gestures system!
at node_modules/react-native-gesture-handler/src/handlers/createHandler.ts:128:4 in showRngh2NoticeIfNeeded
at node_modules/react-native-gesture-handler/src/handlers/createHandler.ts:176:31 in Handler#constructor
at navigation/RootNavigator.js:21:20 in onAuthStateChanged$argument_1
```

**Comment by amandeepmittal** [comment]
Hey @tayfunyasar, thanks for reporting this. Will fix this.

**Comment by amandeepmittal** [comment]
@tayfunyasar, all of the above warnings now should be fixed with PR #72 merged.

Closing this issue.
